client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -75,9 +76,20 @@ export const DetailView: FC = () => {
   useEffect(() => {
     if (encounter)
       setSuffix(
-        `${
-          encounter.document.documentRegistry?.name
-        } - ${dateFormat.localisedDateTime(encounter.startDatetime)}`
+        <>
+          <Breadcrumb
+            to={RouteBuilder.create(AppRoute.Dispensary)
+              .addPart(AppRoute.Patients)
+              .addPart(encounter.patient.id)
+              .build()}
+            key={'patient'}
+          >
+            {`${encounter.patient.firstName} ${encounter.patient.lastName}`}
+          </Breadcrumb>
+          <span key={'encounter'}>{` / ${
```

Comment:
the key isn't required - it was on the original because we were mapping an array. 
Also - since it's just text you can have `key="encounter"`

Reply:
Okay, yeah done that. Removed the unneeded keys, but I had to add a key to the whole thing, as it becomes an item in the Breadcrumb array: 8f194ce78